### PR TITLE
Stojanovic/fe 282 naming issues

### PR DESCRIPTION
**Pull request by Ducica**
related PRS

https://github.com/oarepo/oarepo-ui/pull/254

https://github.com/Narodni-repozitar/nr-model/pull/76

### CODE REVIEW

File: ui/branding/__init__.py [hunk 1]
```diff
@@ -22,7 +22,12 @@ def init_menu():
             "documents.create",
             _("New upload"),
             order=1,
+            # TODO: create function that checks if user has any permissions and as optimalization cache it in session
+            # visible_when=check_permissions,
         )
+        # hide the /admi (maximum recursion depth exceeded menu)
+        admin_menu = current_menu.submenu("settings.admin")
```
Comment (mirekys) [inline ui/branding/__init__.py:29]: Could we move this to oarepo-ui please? As this is broken on every repository app

**Comment by mirekys** [comment]
Version should also be visible on dashboard result list (where it's even more important)
Screenshot 2024-11-15 at 16 31 16

**Comment by Ducica** [comment]
@mirekys ahoj, version input now only exists inside publish request, so unfortunately if your request is just a draft you saved, at that point you wont have where to provide it. Are those published records or just drafts?

File: ui/documents/templates/semantic-ui/documents/Main.jinja [hunk 2]
```diff
@@ -1,5 +1,17 @@
 {#def metadata, ui, record, extra_context #}
-<h1 class="ui header">{{ metadata.title if metadata.title else _("Title not provided") }}</h1>
+{% set ns = namespace(localized_title=None)%}
```
Comment (mirekys) [inline ui/documents/templates/semantic-ui/documents/Main.jinja:2]: `metadata.title` value gets "lost" here, replaced with alternative, never to be seen anywhere, is there a request to try to localize main record title? Shouldn't the main original title always be visible somewhere?

**Review by mirekys — CHANGES_REQUESTED** [review]

**Comment by mirekys** [comment]
> @mirekys ahoj, version input now only exists inside publish request, so unfortunately if your request is just a draft you saved, at that point you wont have where to provide it. Are those published records or just drafts?

Those were drafts with existing publish request, if it later propagates to record metadata and is visible for published records, then it's OK

File: ui/documents/templates/semantic-ui/documents/Main.jinja (hunk 2 above)
Comment (Ducica) [reply ui/documents/templates/semantic-ui/documents/Main.jinja:2]: @mirekys It is OK, that was my interpretation of the request, though I am not 100% sure, if I understood correctly:)

**Comment by Ducica** [comment]
> > @mirekys ahoj, version input now only exists inside publish request, so unfortunately if your request is just a draft you saved, at that point you wont have where to provide it. Are those published records or just drafts?
> 
> Those were drafts with existing publish request, if it later propagates to record metadata and is visible for published records, then it's OK

If it exists in the metadata it will be shown, however, it is another question if this request should work in such a way to attach what you provide when creating request to the draft itself. I think I mentioned it before to Miro, but I am not sure what happened with it in the end.

**Review by mirekys — APPROVED** [review]

**Comment by mirekys** [comment]
We postpone merging this until RDM record based model is merged (it will be clearer how to handle versions)

**Review by mirekys — APPROVED** [review]

**Comment by mirekys** [comment]
> > We postpone merging this until RDM record based model is merged (it will be clearer how to handle versions)
> 
> Ahoj, If the version shown in list item is not OK, I can remove it. I would like to merge this if possible, because it contains other fixes that girls wanted and I don't see that RDM record related things will be implemented for this, upcoming release. Let me know please.

You're right, lets go for a merge then :)

**Pull request merged**